### PR TITLE
Ft/implement siwe signature

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx [hunk 1]
```diff
@@ -66,12 +73,13 @@ export const MissionCTA = ({ CTAs, variableWeeklyAPY }: MissionCtaProps) => {
             lineHeight={{ xs: '14px', md: '18px' }}
             fontWeight={400}
           >
-            Completing any mission below makes you eligible for OP rewards and
-            XP.
+            {!signature &&
```
Comment (admsk2) [inline src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx:76]: what is going to be displayed if signature is true?

**Review by admsk2 — APPROVED** [review]

File: src/hooks/useTurtleMember.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}: UseTurtleProps): UseTurtleRes => {
+  const TURTLE_CHECK_API = `https://points.turtle.club/user/${userAddress}/exists`;
+
+  const { data, isSuccess, isLoading } = useQuery({
+    queryKey: ['turtleMemberCheck'],
+    queryFn: async () => {
+      try {
+        const response = await fetch(TURTLE_CHECK_API);
+        const result = await response.text();
+        console.log(result);
```
Comment (admsk2) [inline src/hooks/useTurtleMember.ts:24]: let's remove that one

File: src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx (hunk 1 above)
Comment (tcheee) [reply src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx:76]: nothing should shown

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**